setup.py

```diff
@@ -36,10 +36,10 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, kh, jt, sdw
 
-VSC_INSTALL_REQ_VERSION = '0.9.19'
+VSC_INSTALL_REQ_VERSION = '0.10'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5',
```

Comment:
so let's just enforce [semantic versioning](http://semver.org/) you mean? I agree, but let's do this for everything then, I'll also start commenting on version numbers in the future :-)